coordinator/src/services/communication/sandbox-spine-client.ts

```diff
@@ -20,6 +20,11 @@ export class SandboxSpineClient implements SpineClient {
           statusCode: 200,
           body: spineResponses.NOMINATED_PRESCRIPTION_RELEASE
         })
+      case hl7V3.Hl7InteractionIdentifier.PATIENT_PRESCRIPTION_RELEASE_REQUEST._attributes.extension:
+        return Promise.resolve({
+          statusCode: 200,
+          body: spineResponses.NOMINATED_PRESCRIPTION_RELEASE
```

Comment:
This will do the job. Might be nice to have a nominated example with 25 prescriptions and a patient example with a single prescription though?

Reply:
agreed, would like to get this in quickly and then improve our examples in next pass